### ISSUE TITLE
Deep Learning for Automatically Detecting Sidewalk Accessibility Problems Using Streetscape Imagery

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#102]
### Links
- Paper: https://dl.acm.org/citation.cfm?id=3353798

### Abstract
- Recent work has applied machine learning methods to automatically find and/or assess pedestrian infrastructure in online map imagery (e.g., satellite photos, streetscape panoramas). While promising, these methods have been limited by two interrelated issues: small training sets and the choice of machine learning model. In this paper, aided by the recently released Project Sidewalk dataset of 300,000+ image-based sidewalk accessibility labels, we present the first examination of deep learning to automatically assess sidewalks in Google Street View (GSV) panoramas. Specifically, we investigate two application areas: automatically validating crowdsourced labels and automatically labeling sidewalk accessibility issues. For both tasks, we introduce and use a residual neural network (ResNet) modified to support both image and non-image (contextual) features (e.g., geography). We present an analysis of performance, the effect of our non-image features and training set size, and cross-city generalizability. Our results significantly improve on prior automated methods and, in some cases, meet or exceed human labeling performance.

- 歩道検出アルゴリズムをトレーニングするために大規模なデータセット（300000枚以上のアノテーション付きデータ）を用意
- ResNetベースでいろんなネットワークを実装して検証
- めちゃくちゃ優秀な検出結果が得られた

<img width="893" alt="スクリーンショット 2019-10-31 15 17 24" src="https://user-images.githubusercontent.com/26996041/67931480-412cc580-fc05-11e9-959e-916bebc0a2b1.png">
<img width="877" alt="スクリーンショット 2019-10-31 15 18 45" src="https://user-images.githubusercontent.com/26996041/67931484-4427b600-fc05-11e9-81a7-c10d1365fe33.png">
